CorsixTH/CorsixTH.lua

```diff
@@ -17,6 +17,12 @@ for _, arg in ipairs({...}) do
   end
 end
 
+-- Add the local luarocks folder to package.cpath.
+local luarocks_path = debug.getinfo(1, "S").source:sub(2):match("(.*/)") ..
+    "luarocks/lib/lua/" .. string.sub(_VERSION,5)
+package.cpath = luarocks_path .. "/?.so;" .. luarocks_path .. "/mime/?.so;" ..
```

Comment:
A little late now, but /mime/?.so and socket/?.so shouldn't have been part of this. ? is replaced by the module name, which is mime.core and socket.core.